src/sender.js

```diff
@@ -155,15 +162,15 @@ class Sender {
     /**
      * Closes the connection to the database. <br>
      * Data sitting in the Sender's buffer will be lost unless flush() is called before close().
+     *
+     * @return {Promise<boolean>} Resolves to true if connection is closed.
```

Comment:
nit: why do we need to return anything here, especially considering that the returned value is always `true`?

Reply:
ok, removing the return value.

Reply:
removed `return true;`, i think it looks better now.